docs/en/observability/synthetics-command-reference.asciidoc

```diff
@@ -83,6 +83,7 @@ npx @elastic/synthetics push --auth <api-key> --url <kibana-url> --project <id|n
 
 `--auth`::
 API key used for {kibana-ref}/api-keys.html[{kib} authentication].
+If you are pushing to a <<private-locations,private location>>, you must use an API key generated in 8.4 or higher.
```

Comment:
@dominiqueclarke is this language accurate?

Reply:
Actually, we may want to be more specific with the requirements, similar to the other PR. For  generating an api key capable of interacting with private locations specifically, users must have write level Kibana permissions for Fleet and Integrations.